src/virtual-wan/azext_vwan/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isPreview": true,
```

Comment:
let's remove this line

Reply:
If next release is stable, `stable` label can be added to re-cal the expected version number